### ISSUE TITLE
PHPStorm freeze when editing Latte template

### ISSUE THREAD

**Issue by MilanPala** [nette-intellij/intellij-latte#117]
I have very slow editing on one template. Other templates (with more lines of code) works fine. When I press Enter for new line there is 4 seconds delay. Whole PHPStorm freeze for this time. Deleting that new line is without problem. I have template class in the same directory for type hinting. 

This is the very truncated template where is still the issue.

Thank you.

```
{templateType \Pd\OrderModule\EshopModule\Presenters\templates\Basket\DefaultTemplate}

{block content}

<article class="js-order">
	<div class="l-in-box">
		<div n:snippet="basketForm" class="js-autorecalc ajax-wrap">
		{if count($basket->getProducts())}

			<form n:name="basketForm">
				<fieldset class="product-list-basket u-mb-0">
					<h1 class="sr-only">
						Košík
					</h1>
					<div class="product-list-basket__list">
						{formContainer items}
							{foreach $basket->getProducts() as $item}
								{continueIf $item instanceof \Pd\OrderModule\EshopModule\Model\Basket\Items\IEvent}
								{formContainer $item->getId()}
											<div class="product-box-basket__head{if $seller} product-box-basket__head--seller{/if}">
												<a href="{plink :Product:Eshop:ItemDetail:default $item->getProductItem()}" class="product-box-basket__link">
													<p class="product-box-basket__img-box img-box img-box--lazy img-box--fixed-ratio">
														{pdImgSource \current($item->getProductItem()->getProduct()->getImages()) 'product-box-w85h85', alt => $item->getName(), class => 'img-box__img', lazy /}
													</p>
												</a>
											</div>
								{/formContainer}
							{/foreach}
						{/formContainer}
					</div>
				</fieldset>
			</form>
		{/if}
		</div>
	</div>
</article>

```

**Comment by stanislav-janu** [comment]
Same problem in ublaboo/datagrid template. Big file === big performance problem.

**Comment by mesour** [comment]
@MilanPala Hello, thanks for reporting, I'm seeing unknown tag `pdImgSource` closed with `/}`. It causes a parse error, but this is another issue. Performance issue will be fixed in future in `1.1.1-RC3`

**Comment by mesour** [comment]
Some performace issues are fixed in: [v1.1.1](https://github.com/nette-intellij/intellij-latte/releases/tag/v1.1.1). If you have still some problem with fixed `pdImgSource`, please reopen issue.
